help/index.php

```diff
@@ -0,0 +1,176 @@
+	<?=common_top()?>
+
+	<meta charset="utf-8">
+	<title>Swamp Servers - Commands</title>
+	<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css">
+	<link href='https://fonts.googleapis.com/css?family=Righteous' rel='stylesheet'>
+	<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
```

Comment:
Remove unused imports

Reply:
for the bootstrap, do I just use the one in you css folder?

Reply:
You don't need to include it, the common_top will include it for you

Reply:
yeah i can remove the font and jquery but the bootstrap i'm using is bootstrap 4 and i'm afraid if the bootstrap provided is 3 so i just thought i would include it